src/main/groovy/com/instabot/operators/weboperations/userextractor/RelatedUsersUpdater.groovy

```diff
@@ -21,6 +24,8 @@ import java.time.LocalDateTime
 import java.time.temporal.ChronoUnit
 
 @Service
+//TODO This class does too many things. You need to separate html processing from business logic.
```

Comment:
I agree on one hand. However, on the other hand, this class has one single feature that needs to be used outside of this class - update the relation (is following/is followed) of the users that have any relationship to the master user.

Now for doing that, I need to have the HTML with Jsoup. For getting the right HTML I need to navigate through the page using Selenium.

The source of my hesitation about splitting Jsoup, Selenium and the process of updating it in database is that they will work only together, and cannot be reused for other purposes. 

To make the code more eye-friendly, I could probably split them into 3 classes, however leaving them in the same package. What do you think?